web/tests/Web.E2ETests/Features/School/FindOrganisation.feature

```diff
@@ -0,0 +1,6 @@
+﻿Feature: Find organisation 
+
+Scenario: Goto school homepage
+	Given I am on find organisation page
```

Comment:
This assume that "school" organisation type has been selected.

Worth considering add and "And" to the "Given"

Reply:
will be done in latest commit.